Sources/SwiftLayoutUtil/ViewPrinter.swift

```diff
@@ -65,9 +65,24 @@ public struct ViewPrinter: CustomStringConvertible {
     /// - Returns: The view itself with the **accessibilityIdentifier** applied
     ///
     @discardableResult
-    public func updateIdentifiers(_ updater: IdentifierUpdater = .nameOnly, rootObject: AnyObject? = nil) -> Self {
-        let viewTags = updater.update(rootObject ?? self.view, viewTags: self.viewTags)
-        return Self.init(self.view, viewTages: viewTags, options: self.options)
+    public func updateIdentifiers<RO>(_ updater: IdentifierUpdater = .nameOnly, rootObject: RO?) -> Self {
```

Comment:
```suggestion
    public func updateIdentifiers<RO>(_ updater: IdentifierUpdater = .nameOnly, rootObject: RO) -> Self {
```

If you take a separate method without the default value, it looks better without the optional.